libraries/AP_EFI/AP_EFI_Serial_Lutan.cpp

```diff
@@ -0,0 +1,95 @@
+/*
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+ 
+#include <AP_HAL/AP_HAL.h>
+#include "AP_EFI_Serial_Lutan.h"
+#include <AP_HAL/utility/sparse-endian.h>
+#include <stdio.h>
+
+#if HAL_EFI_ENABLED
```

Comment:
This should really be `AP_EFI_SERIAL_LUTAN_ENABLED`

Nobody running this is really going to care, but it is a pattern we should spread as widely as we can.